src/NzbDrone.Core/Datastore/Migration/219_nzb_su_url_to_nzb_life.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using FluentMigrator;
+using Newtonsoft.Json.Linq;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(219)]
+    public class nzb_su_url_to_nzb_life : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.Sql("UPDATE \"Indexers\" SET \"Settings\" = replace(\"Settings\", '//api.nzb.su', '//api.nzb.life')" +
+                        "WHERE \"Implementation\" = 'Newznab'" +
+                        "AND \"Settings\" LIKE '%//api.nzb.su%'");
+        }
+    }
+
+    public class IndexerDefinition219
```

Comment:
Since these are just used in the tests I think they'd be better in `219_nzb_su_url_to_nzb_lifeFixture.cs`

Reply:
moved these to internal classes in the test.